ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/cvx.j2

```diff
@@ -0,0 +1,30 @@
+{# CVX #}
+{% if cvx is arista.avd.defined %}
+
+# CVX
+{%     if cvx.shutdown is arista.avd.defined(true) %}
+
+cvx is disabled
+{%     elif cvx.shutdown is arista.avd.defined(false) %}
+
+cvx is enabled
+{%         if cvx.services is arista.avd.defined %}
+
+## CVX services
+
+| Service | Enabled | Password |
+| ------- | ------- | -------- |
+{%             if cvx.services.mcs is arista.avd.defined %}
+{%                 set shut = cvx.services.mcs.shutdown | arista.avd.default('-') %}
```

Comment:
This should probably be something like set enabled  = not cvx.services.mcs.shutdown instead.